.github/workflows/release-packaging.yaml

```diff
@@ -2,6 +2,10 @@ on:
   push:
     branches:
       - main
+    paths:
+      - 'src/**'
+      - '**/Cargo.lock'
+      - '**/Cargo.toml'
```

Comment:
@michaelclapham Could we use an approach like this to disable checks for documentation-only changes?